json-modules/json/src/test/java/com/baeldung/jsonjava/JSONObjectIntegrationTest.java

```diff
@@ -22,7 +23,29 @@ public void givenJSONJava_thenCreateNewJSONObject() {
     }
 
     @Test
-    public void givenMapObject_thenCreateJSONObject() {
+    void givenJSON_whenParsed_thenCorrectValueReturned() {
+        String jsonString = """
+                            {
+                                "type": "Feature", 
+                                "geometry": "Point",                          
+                                "properties": {
+                                                  "isValid": true, 
+                                                  "name": "Sample Point"
+                                              }
+                            }
+                            """;
+        JSONObject jsonObject = new JSONObject(jsonString);
+        String type = jsonObject.getString("type");
+        String geometry = jsonObject.getString("geometry");
+        JSONObject properties = jsonObject.getJSONObject("properties");
+        boolean isValid = properties.getBoolean("isValid");
+        assertEquals(type,"Feature");
+        assertEquals(geometry,"Point");
```

Comment:
Please reverse the order of the arguments. It's intended to be assertEquals(expected, actual)

Reply:
Reversed